src/index.js

```diff
@@ -20,10 +19,59 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 0];
+const operators = ['+', '-', '*', '/', '='];
+
+const init = {
+  number: 0,
+  temporary: 0,
+  operator: '',
+};
+
+const calculation = (operator, firstNumber, secondNumber) => {
```

Comment:
```suggestion
const calculation = (operator, a, b) => {
```

단순하게 위와 같이 표현해도 좋을 것 같아요.